app/components/questionnarie/Questionnarie.tsx

```diff
@@ -49,13 +74,20 @@ const Questionnaire = ({ questions, options, quizLogic }: QuestionnarieProps) =>
         </div>
       ) : (
         <form onSubmit={handleSubmit}>
-          <div className={styles.question}>
-            <label aria-label="Question" className={styles.question_label}>{`${
+          <fieldset className={styles.question}>
+            <legend id="question-label" className={styles.question_label}>{`${
               currentQuestionIndex + 1
-            }: ${questions[currentQuestionIndex]}`}</label>
-            <div className={styles.options}>
+            }: ${questions[currentQuestionIndex]}`}</legend>
+            <div role="group" aria-labelledby="question-label" className={styles.options}>
```

Comment:
Could you validate if this div can be ul ?

Reply:
double checked with chatGpt

"From a semantic standpoint, the use of a <div> here is appropriate and clean. The <div> element is being used to group a set of related controls (the radio buttons), which is exactly what the <div> element is for - it represents a division or a section in an HTML document."

"So to sum up, your code looks good from a semantic standpoint, and there's no clear benefit to using a <ul> instead of a <div> in this context. As long as the styling works as intended and the UI is user-friendly and accessible, you're on the right track."